test/refactoring/ConvertAsync/convert_async_attributed_renames.swift

```diff
@@ -0,0 +1,95 @@
+// REQUIRES: concurrency
+
+// RUN: %empty-directory(%t)
+
+@available(*, renamed: "simple2")
+func simple(_ completion: @escaping (String) -> Void) { }
+@available(*, renamed: "simple2")
+func nonCompletionName(_ random: @escaping (String) -> Void) { }
+func simple2() async -> String { }
+
+// RUN: %refactor-check-compiles -convert-to-async -dump-text -source-filename %s -pos=%(line+1):6 | %FileCheck -check-prefix=SIMPLERENAMED %s
+func simpleRenamed() {
+  // preserve me
+  simple { str in
+    print(str)
+  }
+  // and me
+  nonCompletionName { str in
+    print(str)
+  }
+}
+// SIMPLERENAMED: func simpleRenamed() async {
+// SIMPLERENAMED-NEXT: // preserve me
+// SIMPLERENAMED-NEXT: let str = await simple2()
+// SIMPLERENAMED-NEXT: print(str)
+// SIMPLERENAMED-NEXT: // and me
+// SIMPLERENAMED-NEXT: let str1 = await simple2()
+// SIMPLERENAMED-NEXT: print(str1)
```

Comment:
Wouldn’t it make sense to also add a check
```
// SIMPLERENAMED-NEXT: }
```
to verify that the function really ends here? Same for all the other test cases.

Reply:
Sure